test/e2e/framework/volume_util.go

```diff
@@ -492,3 +492,15 @@ func InjectHtml(client clientset.Interface, config VolumeTestConfig, volume v1.V
 	err = WaitForPodSuccessInNamespace(client, injectPod.Name, injectPod.Namespace)
 	Expect(err).NotTo(HaveOccurred())
 }
+
+func CreateGCEVolume() (*v1.PersistentVolumeSource, string) {
```

Comment:
Just fyi, [_persistent_volumes-gce.go#L86_](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/persistent_volumes-gce.go#L86) could now call this func.